tvb/recon/cli/core.py

```diff
@@ -31,8 +31,6 @@ def __new__(cls: type, name: str, bases: Tuple[type],
                 attrs: Dict[str, object]) -> type:
         for key, value in list(attrs.items()):
             if not key.startswith('_'):
-                value:
-                    str
```

Comment:
Aha, on one line it makes sense (this two line syntax is invalid under 3.6, at least according to the interpreter). I'll put it back.